CHANGELOG.md

```diff
@@ -39,111 +37,141 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
-## [v0.3.3](https://github.com/cyberark/summon/releases/tag/v0.3.3) - 2016-02-29
+## [0.3.3] - 2016-02-29
 ### Fixed
-* Now fails more gracefully on unknown flags
+- Now fails more gracefully on unknown flags
 
-## [v0.3.2](https://github.com/cyberark/summon/releases/tag/v0.3.2) - 2016-02-10
+## [0.3.2] - 2016-02-10
 ### Fixed
-* `@SUMMONENVFILE` is now ensured to contain a trailing newline [GH-22](https://github.com/cyberark/summon/issues/22)
+- `@SUMMONENVFILE` is now ensured to contain a trailing newline [GH-22](https://github.com/cyberark/summon/issues/22)
 
-## [v0.3.1](https://github.com/cyberark/summon/releases/tag/v0.3.1) - 2016-02-03
+## [0.3.1] - 2016-02-03
 ### Fixed
-* Integer values set in secrets.yml are now parsed correctly [GH-21](https://github.com/cyberark/summon/issues/21)
+- Integer values set in secrets.yml are now parsed correctly [GH-21](https://github.com/cyberark/summon/issues/21)
 
-## [v0.3.0](https://github.com/cyberark/summon/releases/tag/v0.3.0) - 2015-08-21
+## [0.3.0] - 2015-08-21
 ### Changed
-* Install bash completions if available
-* Switch to tar.gz instead of .zip
-* Try to find provider in the default path if just a name given
-* Allow -p argument to override SUMMON_PROVIDER envar
-* Check if provider exists and is executable
+- Install bash completions if available
+- Switch to tar.gz instead of .zip
+- Try to find provider in the default path if just a name given
+- Allow -p argument to override SUMMON_PROVIDER envar
+- Check if provider exists and is executable
 
-## [v0.2.1](https://github.com/cyberark/summon/releases/tag/v0.2.1) - 2015-06-29
+## [0.2.1] - 2015-06-29
 ### Fixed
-* Improve provider path handling [5df0fde](https://github.com/cyberark/summon/commit/5df0fdeb182884371ad647d0a9493a5e07d3e0e4)
+- Improve provider path handling [5df0fde](https://github.com/cyberark/summon/commit/5df0fdeb182884371ad647d0a9493a5e07d3e0e4)
 
-## [v0.2.0](https://github.com/cyberark/summon/releases/tag/v0.2.0) - 2015-06-23
+## [0.2.0] - 2015-06-23
 ### Added
-* `@SUMMONENVFILE` for better Docker integration
+- `@SUMMONENVFILE` for better Docker integration
+
 ### Fixed
-* -D variable interpolation now plays nicely with the shell
+- -D variable interpolation now plays nicely with the shell
 
-## [v0.1.2](https://github.com/cyberark/summon/releases/tag/v0.1.2) - 2015-06-10
+## [0.1.2] - 2015-06-10
 ### Fixed
-* Fix --help and --version flags
+- Fix --help and --version flags
+
 ### Changed
-* Vendor dependencies with Godep
+- Vendor dependencies with Godep
 
-## [v0.1.1](https://github.com/cyberark/summon/releases/tag/v0.1.1) - 2015-06-09
+## [0.1.1] - 2015-06-09
 ### Changed
-* Attach stdin to allow running interactive processes wrapped with summon
-* Changed name from 'cauldron' to 'summon'
+- Attach stdin to allow running interactive processes wrapped with summon
+- Changed name from 'cauldron' to 'summon'
 
-## [v0.1.0](https://github.com/cyberark/summon/releases/tag/v0.1.0) - 2015-06-03
+## 0.1.0 - 2015-06-03
```

Comment:
Should be able to use hyperlink to [0.1.0] (initial release)

Reply:
Fixed